Cargo.toml

```diff
@@ -10,12 +10,12 @@ fuel-types = { version = "0.3", features = ["serde-types"] }
 fuel-vm = { version = "0.11", features = ["serde"] }
 serde_json = "1.0"
 shellfish = { version = "0.6.0", features = ["rustyline", "async", "tokio"] }
-surf = "2.0.0"
+surf = "2.3.2"
```

Comment:
Not sure why we are using major/minor/patch here instead of just major/minor

Reply:
Me neither. Moved to just major.minor.